templates/apollo-graphql-react/files/back/src/resolvers/query.js

```diff
@@ -0,0 +1,55 @@
+import fetch from "node-fetch";
+
+const {
+  YOUTUBE_API_KEY = "<EXPORT YOUTUBE_API_KEY WITH YOUR KEY>",
+} = process.env;
```

Comment:
If I read this correctly, it's "set the YOUTUBE_API_KEY const to the matching value out of process.env, and if that's not set then use this default message string instead."  Right?  (Just making sure my JS knowledge is up to date here.)

Reply:
Yup!